benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -659,8 +679,12 @@ void GraphicsBenchmarkApp::RecordCommandBuffer(PerFrame& frame, grfx::SwapchainP
 
     // Record commands for the scene using one renderpass
     frame.cmd->BeginRenderPass(currentRenderPass);
```

Comment:
If both the skybox and the sphere are disabled, I think this renderpass should not be begin/ended?